packages/eas-cli/src/update/configure.ts

```diff
@@ -96,6 +97,10 @@ async function ensureEASUpdatesIsConfiguredInExpoConfigAsync({
 }): Promise<{ projectChanged: boolean; exp: ExpoConfig }> {
   const modifyConfig: Partial<ExpoConfig> = {};
 
+  if (isDefinitelyUsingClassicUpdates(exp)) {
+    modifyConfig.updates = { useClassicUpdates: false };
```

Comment:
I think we want to be careful about modifying it for them, as they could forget to change it back in the case that they miss the message that we changed it for them. Maybe I'm overthinking this though.

The main case I can think of people needing both is to publish hotfixes. In this case though, they would probably need to create a sort of hotfix branch and have a diverged config, so I guess modifying it for them is okay?

Fine to leave this as-is I think, just want to make sure we're thinking through the edge cases as far as using both goes.

Reply:
I put up an alternative PR here that doesn't modify the app config: https://github.com/expo/eas-cli/pull/1915. Regardless of the approach I think it is better to remove "useClassicUpdates" than to set it to false since the field is going away in SDK 50.

Reply:
The other major point of configuration is `updates.url`. `update:configure` and `update` both set the `updates.url` to be the EAS Update one. In the case of using both Classic and EAS Update, the user will need to maintain a diverged config.